src/patternfly/components/DataList/data-list-cell.hbs

```diff
@@ -1,6 +1,9 @@
-<div class="{{pfv}}data-list__cell{{#if data-list-cell--modifier}} {{data-list-cell--modifier}}{{/if}}"
+<div class="{{pfv}}data-list__cell{{#if data-list-cell--modifier}} {{data-list-cell--modifier}}{{/if}}{{#if data-list-cell--IsTruncate}} pf-m-truncate{{/if}}"
```

Comment:
Nit - do you mind reversing these? Most/all of the hbs helpers have a `--modifier` to add extra classes and `--attribute` to add extra attributes to the tag, and those are added at the end of the classes and attributes.

```suggestion
<div class="{{pfv}}data-list__cell{{#if data-list-cell--IsTruncate}} pf-m-truncate{{/if}}{{#if data-list-cell--modifier}} {{data-list-cell--modifier}}{{/if}}"
```